internalshared/configutil/kms.go

```diff
@@ -220,6 +220,9 @@ func configureWrapper(configKMS *KMS, infoKeys *[]string, info *map[string]strin
 	if configKMS.Type == wrapping.WrapperTypeTransit.String() {
 		mergeTransitConfig(configKMS.Config, envConfig)
 	} else {
+		if configKMS.Config == nil {
```

Comment:
I think this should be outside the `if` statement, because the transit seals also use the map

Reply:
done